packages/notifier/src/asyncIterableAdaptor.js

```diff
@@ -103,3 +103,35 @@ export const observeIteration = (asyncIterableP, iterationObserver) => {
  */
 export const observeNotifier = (notifierP, iterationObserver) =>
   observeIteration(subscribeLatest(notifierP), iterationObserver);
+
+/**
+ * Watch an ephemeral notifier, and recover if it fails due to upgrade or other
+ * reasons. Since it's expected to be a perpetual notifier, the watcher's
```

Comment:
what is a "perpetual notifier"? I don't see such a thing. Do you mean one that never finishes? I'm surprised if we don't already have a term for that but I can't think of it.

Why do we need to distinguish that here?

Reply:
I came up with that. I mean a notifier that isn't expected to fail or finish, unless there's an upgrade. This supports that by providing a straightforward way to get a replacement if it breaks.

@mhofman suggested adding it here, when I would have otherwise needed a generic place reachable from vaultFactory. We initially thought it would also be used by auctions, but the case there is slightly different.

Reply:
I have nothing against the suggestion, but I don't remember making it.

Reply:
I'm very reluctant to introduce this concept without some wholistic design and documentation. The notifier package is already an amalgamation of loosely coherent concepts.

Since this doesn't need to be in the notifier package, please put it only where it's needed and we can revisit later making it a more general capability. 

When you do move it, please update the jsdoc with the clear explanation you had in this thread. I.e. instead of introducing (without definition) the term "perpetual", just say approx, "[It] isn't expected to fail or finish, unless there's an upgrade. This supports that by providing a straightforward way to get a replacement if it breaks."

Reply:
done